docs/docs/features/caches.md

```
@@ -17,7 +13,7 @@ A scrollable list of all configured `Caches` in the application, grouped by thei
 
 ---
 
-## The Cache Details{#details}
+## The Cache Details
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)